projects/remote_transceiver/src/remote_transceiver.cpp

```diff
@@ -62,12 +62,12 @@ HTTPServer::HTTPServer(tcp::socket socket, SailbotDB & db) : socket_(std::move(s
 
 void HTTPServer::doAccept() { readReq(); }
 
-Listener::Listener(bio::io_context & io, tcp::acceptor acceptor, SailbotDB && db)
-: io_(io), acceptor_(std::move(acceptor)), db_(std::move(db)){};
+Listener::Listener(tcp::acceptor acceptor, SailbotDB && db, bio::strand<bio::io_context::executor_type> strand)
+: acceptor_(std::move(acceptor)), db_(std::move(db)), strand_(std::move(strand)){};
 
 void Listener::run()
 {
-    acceptor_.async_accept(bio::make_strand(io_), [&](beast::error_code e, tcp::socket socket) {
+    acceptor_.async_accept(strand_, [&](beast::error_code e, tcp::socket socket) {
```

Comment:
Again, I think we should just do what's done in the [official example](https://github.com/boostorg/beast/blob/f8ff44d072803e8c193946818ee7d239c5834b10/example/http/server/async/http_server_async.cpp#L393) and make a new strand for each request. That way, if we need to use io_context in the future, we won't have to worry about synchronization after accepting a request.

Reply:
listener now creates a new strand on new connection which is managed by main acceptor strand